lib/metrics/ZenkoMetrics.ts

```diff
@@ -1,26 +1,23 @@
 import promClient from 'prom-client';
 
-const collectDefaultMetricsIntervalMs =
-    process.env.COLLECT_DEFAULT_METRICS_INTERVAL_MS !== undefined ?
-        Number.parseInt(process.env.COLLECT_DEFAULT_METRICS_INTERVAL_MS, 10) :
-        10000;
-
-promClient.collectDefaultMetrics({ timeout: collectDefaultMetricsIntervalMs });
+// 'timeout' property is not needed/supported
+// https://github.com/siimon/prom-client/blob/199b7d19f8c8c34ee8653264e8dc0e57b420074f/CHANGELOG.md#1200---2020-02-20
+promClient.collectDefaultMetrics();
 
 export default class ZenkoMetrics {
-    static createCounter(params: promClient.CounterConfiguration) {
+    static createCounter(params: any) {
```

Comment:
`unknown` might be more appropriate if type info is not available; why the widening from `CounterConfiguration` to `any`?

Reply:
The definition has changed in the newer version, I wasn't sure, so used `any`. Modifying it.